tests/e2e/BarChart/BarChart.spec.ts

```diff
@@ -13,18 +13,19 @@ describe("Bar chart", () => {
     });
 
     it("should be generated with two traces", () => {
-        chart.traces.waitForValue();
-
-        expect(chart.traces.value.length).toBe(2);
+        browser.waitUntil(() => {
+            return chart.traces.map((elem) => elem.isDisplayed()).length > 1;
+        });
+        expect(chart.traces.map((elem) => elem.isDisplayed()).length).toBe(2);
     });
 
     // with other drivers apart from chrome it does not autoscroll to the element in overflow:auto block
-    it("should hide a bar serie when a serie toggle item is clicked", () => {
+    xit("should hide a bar serie when a serie toggle item is clicked", () => {
```

Comment:
What is the future plan for this test case? If we are never going to be able to test this with Firefox, it's better to remove it. Otherwise I would suggest to create a todo comment and add a story to the backlog.

Reply:
Yeah! That's the idea, I'm going to create a new story for that. I'll follow your suggestion and add this comment as TO DO as well. Thanks!